service-python/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+
+def lib_queues(channel):
```

Comment:
Can you give more details as to what this file is doing

Reply:
Sure, because the main_consumer pulls in the `lib` from whichever service folder it is copied into, to test it the `lib` has to be mocked.

Reply:
Yeah this is a new way that I found to mock modules that are used in the tests.